src/components/dialogs/limits/limits-pane-utils.js

```diff
@@ -21,42 +27,74 @@ import { areArrayElementsUnique, formatTemporaryLimits } from 'components/utils/
 import yup from 'components/utils/yup-config';
 import { isNodeBuilt } from '../../graph/util/model-functions';
 
-const temporaryLimitsTableValidationSchema = () => ({
+const limitsGroupValidationSchema = () => ({
+    [ID]: yup.string().nonNullable().required(),
+    [CURRENT_LIMITS]: yup.object().shape(currentLimitsValidationSchema()),
+});
+
+const temporaryLimitsValidationSchema = () => {
+    return yup.object().shape({
+        [TEMPORARY_LIMIT_DURATION]: yup.number().nullable().min(0),
+        [TEMPORARY_LIMIT_VALUE]: yup.number().nullable().positive(),
+        [TEMPORARY_LIMIT_NAME]: yup
+            .string()
+            .nullable()
+            .when([TEMPORARY_LIMIT_VALUE, TEMPORARY_LIMIT_DURATION], {
+                is: (limitValue, limitDuration) => limitValue || limitDuration,
+                then: () => yup.string().nullable().required(),
+                otherwise: () => yup.string().nullable(), // empty lines are ignored
+            }),
+    });
+};
+
+const currentLimitsValidationSchema = () => ({
     [PERMANENT_LIMIT]: yup.number().nullable().positive('permanentCurrentLimitMustBeGreaterThanZero'),
     [TEMPORARY_LIMITS]: yup
         .array()
-        .of(
-            yup.object().shape({
-                [TEMPORARY_LIMIT_NAME]: yup.string().required(),
-                [TEMPORARY_LIMIT_DURATION]: yup.number().nullable().min(0),
-                [TEMPORARY_LIMIT_VALUE]: yup.number().nullable().positive(),
-            })
-        )
+        .of(temporaryLimitsValidationSchema())
         .test('distinctNames', 'TemporaryLimitNameUnicityError', (array) => {
             const namesArray = array
                 .filter((l) => !!l[TEMPORARY_LIMIT_NAME])
                 .map((l) => sanitizeString(l[TEMPORARY_LIMIT_NAME]));
             return areArrayElementsUnique(namesArray);
         })
         .test('distinctDurations', 'TemporaryLimitDurationUnicityError', (array) => {
-            const durationsArray = array.map((l) => l[TEMPORARY_LIMIT_DURATION]);
+            const durationsArray = array.map((l) => l[TEMPORARY_LIMIT_DURATION]).filter((d) => d); // empty lines are ignored
             return areArrayElementsUnique(durationsArray);
         }),
 });
 
-const limitsValidationSchema = (id) => ({
-    [id]: yup.object().shape({
-        [CURRENT_LIMITS_1]: yup.object().shape(temporaryLimitsTableValidationSchema()),
-        [CURRENT_LIMITS_2]: yup.object().shape(temporaryLimitsTableValidationSchema()),
-    }),
-});
+const limitsValidationSchema = (id, onlySelectedLimits = true) => {
```

Comment:
limitsValidationSchema is called only in  getLimitsValidationSchema = (onlySelectedLimits = true, id = LIMITS)

```suggestion
const limitsValidationSchema = (id, onlySelectedLimits) 
```

Reply:
Indeed. Changed.